docs/PacketFence_Developers_Guide.asciidoc

```diff
@@ -255,7 +255,7 @@ is used by the `pfcmd` CLI) with:
 VLAN assignment
 ~~~~~~~~~~~~~~~
 
-pfsetvlan uses the `getNormalVlan` function defined in `pf::vlan::custom`
+pfsetvlan uses the `getNormalVlan` function defined in `pf::role::custom`
```

Comment:
I think we should rename everything to "getDefaultRole".

We create the "default" role when installing PacketFence - why call it later "normal"?